css/main.css

```diff
@@ -5,6 +5,8 @@
     --button-bg: rgb(255, 255, 255);
     --primary: rgb(26,115,232);
     --primary-dark: rgb(25,103,210);
+    --secondary-variant: rgb(210,227,252);
+    --on-secondary-variant: rgb(24,90,188);
```

Comment:
what does "on" stand for?
Also, "secondary" is enough, no need for "variant"

Reply:
Copied css straight out of google calendar without thinking, bad practice obviously